gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/src/test/java/io/gravitee/gateway/standalone/container/GatewayTestContainer.java

```diff
@@ -24,6 +24,8 @@
 import io.gravitee.node.api.cache.CacheManager;
 import io.gravitee.node.api.cluster.ClusterManager;
 import io.gravitee.node.container.NodeFactory;
+import io.gravitee.node.monitoring.handler.NodeMonitoringEventHandler;
```

Comment:
Can be removed?

Reply:
It was causing issues with Spring so I removed it. Now spring issues have been resolved (it was not related to this bean in particular), but it was still unsed in tests, all related services were removed too from the container, so it's fair to remove it too, it make sense so that the whole monitoring stack is out of scope. That being said and to be fair I don't why it was removed in the first place. Maybe @ytvnr  knows about this.